### PR TITLE
P0 41 Button

### PR DESCRIPTION
**Pull request by tomaszdanczak**

### CODE REVIEW

File: src/components/atoms/Button/Button.styles.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+export const StyledButton = styled.button`
+  border: none;
+  border-radius: 25px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+  font-weight: bold;
+  line-height: 15px;
+  font-family: 'Spartan', sans-serif;
```
Comment (LachPawel) [inline src/components/atoms/Button/Button.styles.js:12]: Note: Once we merge the branch with typography, you can use the utils here.

File: src/components/atoms/Button/Button.styles.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      case 'buttonFourDark':
+        return '#373B53';
+      case 'buttonFive':
+        return '#EC5757';
+      case 'buttonSix':
+        return '#F9FAFE';
+
+      default:
+        return '#FFFFFF';
+    }
```
Comment (LachPawel) [inline src/components/atoms/Button/Button.styles.js:40]: Note: All of the color values can be taken from the utils once they are merged, just like above. However, for the time being it's OK :)

**Review by LachPawel — APPROVED** [review]
Overall, looks good congratulations :+1:  Especially storybook part is great :) Maybe you can just wait for previous task to get merged, pull the changes and adjust those details.

**Comment by tomaszdanczak** [comment]
Note: I will do some improvements in future (I will use variables and add tests)

**Pull request merged**